Snake/SnakeTest/Database_Test.cs

```diff
@@ -1,56 +1,68 @@
-﻿using System;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
 using DatabaseLayer;
+using BusinessLayer;
 using System.IO;
+using System.Collections.Generic;
+using System;
 
 namespace SnakeTest
 {
     [TestClass]
     public class Database_Test
     {
+        private const string testMazeLevel = "DBTest";
         [TestMethod]
         public void getHighSCore_Database_Will_Write_Score()
         {
-            Database db = new Database();
-            int sampleHighScore = 30;
+            Database db = new Database(testMazeLevel);
+            HighScore highSCore = new HighScore();
+            highSCore.score = 10;
+            highSCore.name = "Test_DB_Will_Write_Score";
+            List<HighScore> highScoreList = new List<HighScore>();
 
-            bool checkCompleted = db.setHighScore(sampleHighScore);
+            bool checkCompleted = db.setHighScore(highScoreList);
 
             Assert.IsTrue(checkCompleted, "Score not written");
         }
 
         [TestMethod]
         public void setHighSCore_Database_Will_Store_Score()
         {
-            Database db = new Database();
-            int highScore;
-            int sampleHighScore = 30;
-            db.setHighScore(sampleHighScore);
+            Database db = new Database(testMazeLevel);

```

Comment:
Testing actual database calls is usually not a great idea - means that the database has to be deployed somewhere where the tests run, and be kept up to date.

For the purposes of your project and learning, its probably OK though.